wgsl/index.bs

```diff
@@ -2168,9 +2168,9 @@ Proposal: pipeline creation fails with an error.
 
 <div class='example wgsl global-scope' heading='Module constants, pipeline-overrideable'>
   <xmp>
-    [[constant_id(0)]]    const has_point_light : bool = true;      // Algorithmic control
-    [[constant_id(1200)]] const specular_param : f32 = 2.3;         // Numeric control
-    [[constant_id(1300)]] const gain : f32;                         // Must be overridden
+    [[constant_id(0)]]    let has_point_light : bool = true;      // Algorithmic control
```

Comment:
I'm not sure about `let` at global scope. `const` works good there, and it matches the "constant_id" semantics. It's especially cringy in places where no values are provided.

We probably don't want to talk about "specialization lets", we want "specialization constants".

Previously, we expressed concerns about `const` at local scope meaning a different thing from `const` at module scope (cc @kainino0x ), so now is our chance to reflect the difference in naming.

Reply:
Discussed at 2021-03-30 meeting: use 'let' at module-scope.
A 'let' is how you declare a constnat.